### DISCUSSION TITLE
Reuse the empty space while column grouping

### DISCUSSION THREAD

**Discussion by KaushikEDS** [IgniteUI/igniteui-angular#14114] [Q&A] [answered]
In the current design, the column grouping is represented by a blue line and the column summaries by a yellow line. There is a noticeable amount of unused space within the blue line area that could potentially be used more efficiently. The attached screenshot illustrates this issue.

![image](https://github.com/IgniteUI/igniteui-angular/assets/79090369/453d696b-bb81-40d3-9918-f5281a6c4b79)

Basically our clients like to see their data in a very concise manner, so any suggestions regarding this?

**Comment by ddaribo** [comment, marked as answer]
Hi @KaushikEDS,,

The Group row can be templated as described in the [“Group Row Templates” section](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/groupby#group-row-templates) of the IgxGrid Group By topic in our docs. So, any custom content can be placed there. Although, I would say that this is a requirement that is to be handled on application level and is not quite a candidate for a general feature request about the `IgxGrid`.
